### ISSUE TITLE
Preprocess Dataset

### ISSUE THREAD

**Issue by atanumandal0491** [TensorSpeech/TensorFlowASR#198]
Hi, @usimarit 
In 'datasets/asr_dataset.py' line 141 u called line 41 of 'augmentations/augmentation.py', which is calling 
self.signal_augmentations = self.parse(config.pop("signal_augment", {}))
self.feature_augmentations = self.parse(config.pop("feature_augment", {}))
In config file no dict of signal_augment is present, whereas feature_augment is showing None, results in outputs as None in sub-class '_augment'. 

Can you guide me?

**Comment by usimarit** [comment]
@atanumandal0491 Since version >1.0, I removed the predefined signal augmentations (I prefer augmentations implemented in tensorflow rather than nlpaug - to avoid the use of tf.numpy_function for TPU compatibility). If you want to use signal augmentations, you'll have to code them yourself.
And the feature augmentations methods currently "predefined" are time masking and frequency masking in specaugment.

Showing the error messages is better for me to know what is happening in your case.

**Comment by atanumandal0491** [comment]
Hi, I was able to solve the problem. But yet on training using RnntLoss, I am getting "inf" as training loss. Unable to find the solution.

**Comment by usimarit** [comment]
@atanumandal0491 Please tell me more about what rnnt loss you are using (warp rnnt, tf rnnt)? What language you are training? What type of text decoding you are using (characters, subwords, sentence piece)? What version you are using?

**Comment by atanumandal0491** [comment]
Hi, I did a mail to you.
